sklearn/preprocessing/tests/test_data.py

```diff
@@ -564,6 +566,52 @@ def test_robust_scaler_zero_variance_features():
     assert_array_almost_equal(X_trans_new, X_expected_new, decimal=3)
 
 
+def test_maxabs_scaler_zero_variance_features():
```

Comment:
can you please add a check on sparse data. Maybe within the same test just do sp.csr_matrix(X) or something?